br/pkg/lightning/backend/local/localhelper.go

```diff
@@ -410,10 +410,8 @@ func (local *Backend) BatchSplitRegions(
 			// Wait for a while until the regions successfully splits.
 			ok, err2 := local.hasRegion(ctx, region.Region.Id)
 			if !ok || err2 != nil {
-				if err2 == nil {
```

Comment:
why remove this branch?

Reply:
it will print lots of log, each line for one region, so I removed this branch. For `err2 == nil` case we still can get the total failed number of regions in line 428